clients/cli/src/prover.rs

```diff
@@ -70,28 +72,30 @@ fn get_file_as_byte_vec(filename: &str) -> Vec<u8> {
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    // Configure the tracing subscriber
+    //
+    // 1. INITIALIZATION AND SETUP
```

Comment:
This is confusing with 1. INITIAL SETUP below. Maybe remove one of the comments?